Error in "Check status of specified services" during Ops Server installation

I added functionality within the StartStopServices.py to check if the status of the service before trying to request the start or stop of the service in case it's already at that requested state. When I run the Ops Server installation process it throws the following error:
# 

Start the following services:
   Utilities//Geometry.GeometryServer,Utilities//PrintingTools.GPServer

---
- Check status of specified services...

****\* ERROR ENCOUNTERED *****
PYTHON ERRORS:
Traceback info:
  File "C:\ops-server-config\Publish\Server\StartStopServices.py", line 162, in <module>
    serviceStatus = getServiceStatus(serverName, serverPort, userName, passWord, folder, serviceNameAndType)

Error Info:
[Errno socket error] [Errno 1] _ssl.c:504: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol

--- a/Publish/Server/StartStopServices.py
+++ b/Publish/Server/StartStopServices.py
@@ -159,7 +159,7 @@ try:
                 folder = service.split("/")[0]
                 serviceNameAndType = service.split("/")[1]
             
-            serviceStatus = getServiceStatus(serverName, serverPort, userName, passWord, folder, serviceNameAndType)
+            serviceStatus = getServiceStatus(serverName, serverPort, userName, passWord, folder, serviceNameAndType, useSSL)
             realTimeState = serviceStatus.get("realTimeState")
             if realTimeState:
                 if realTimeState.upper() == actionStatusMap[serviceAction.upper()]:

